policies.go

```diff
@@ -651,7 +651,6 @@ func (t *tokenAwareHostPolicy) Pick(qry ExecutableQuery) NextHost {
 	var replicas []*HostInfo
 
 	if qry.GetSession() != nil && qry.GetSession().tabletsRoutingV1 {
-		t.tablets.mu.Lock()
```

Comment:
Not sure why this is here, the commit message is about removing Conn.mu locking

Reply:
I think it was wierd merge issue, it is gone after rebase